sagemaker_build_and_push.sh

```diff
@@ -0,0 +1,34 @@
+%%sh
+
+# The name of our algorithm/ECR repo
+algorithm_name=grants_tagger
+
+#make serve executable
+chmod +x src/serve
+
+account=$(aws sts get-caller-identity --query Account --output text)
+
+# Get the region defined in the current configuration (default to us-west-2 if none defined)
+region=$(aws configure get region)
+region=${region:-eu-west-1}
+
+fullname="${account}.dkr.ecr.${region}.amazonaws.com/${algorithm_name}:latest"
+
+# If the repository doesn't exist in ECR, create it.
+aws ecr describe-repositories --repository-names "${algorithm_name}" > /dev/null 2>&1
+
+if [ $? -ne 0 ]
+then
+    aws ecr create-repository --repository-name "${algorithm_name}" > /dev/null
+fi
+
```

Comment:
pushing can be relatively simple, it can also be added in the `Makefile`. This is the code from the blog
![Uploading Screenshot 2023-04-03 at 11.00.52 AM.png…]()

Reply:
the main simplification here is that I take the region and account_id from environment variables